tomster-player/app/routes/albums.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+import _ from 'lodash/lodash';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+const { RSVP, inject: { service } } = Ember;
+
+export default Ember.Route.extend(AuthenticatedRouteMixin, {
+  commentUpdates: service(),
+
+  model() {
+    return this.get('store').findAll('album').then((albums) => {
+      const relationPromises = _.union(
+        albums.invoke('get', 'songs'),
+        albums.invoke('get', 'comments')
+      );
```

Comment:
Just curious, why is `_.union` used here? Is the uniqueness somewhat important? And since we are dealing with different types within the arrays, uniqueness doesn't make much sense here, right?

Maybe `_.flatten` should be used here?